app/upgrades/v8/upgrades.go

```diff
@@ -3,29 +3,73 @@ package v8
 import (
 	"errors"
 
+	"github.com/cosmos/cosmos-sdk/store/prefix"
 	sdk "github.com/cosmos/cosmos-sdk/types"
 	"github.com/cosmos/cosmos-sdk/types/module"
+	banktypes "github.com/cosmos/cosmos-sdk/x/bank/types"
 	upgradetypes "github.com/cosmos/cosmos-sdk/x/upgrade/types"
 	icacontrollertypes "github.com/cosmos/ibc-go/v5/modules/apps/27-interchain-accounts/controller/types"
 	icahosttypes "github.com/cosmos/ibc-go/v5/modules/apps/27-interchain-accounts/host/types"
 
 	"github.com/cosmos/gaia/v8/app/keepers"
 )
 
+func fixBankMetadata(ctx sdk.Context, keepers *keepers.AppKeepers) error {
```

Comment:
If it is not too hard, maybe we can add some tests to this method

Reply:
added a test